ttnn/cpp/ttnn/operations/data_movement/expand/device/expand_device_operation.cpp

```diff
@@ -0,0 +1,83 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "expand_device_operation.hpp"
+
+#include <cstdint>
+
+#include "ttnn/operations/core/compute_kernel/compute_kernel_config.hpp"
```

Comment:
It seems like compute_kernel_config is not used anywhere in this operation. You should either use it in the compute kernel or remove the include. I’d prefer if you support compute_kernel_config in this operation.

Reply:
Done